splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/table/TableExpansionRow.jsx

```diff
@@ -4,34 +4,32 @@ import Table from '@splunk/react-ui/Table';
 import { _ } from '@splunk/ui-utils/i18n';
 
 function getExpansionRowData(row, moreInfo) {
-    return (
-        moreInfo &&
-        moreInfo.length &&
+    const DefinitionLists = [];
+    if (moreInfo?.length) {
         moreInfo.map((val) => {
             const label = _(val.label);
-            return (
-                <>
-                    {row[val.field] && (
-                        <>
-                            <DL.Term>{label}</DL.Term>
-                            <DL.Description>
-                                {val.field === 'disabled'
-                                    ? val.mapping[row[val.field]]
-                                    : `${row[val.field]}`}
-                            </DL.Description>
-                        </>
-                    )}
-                </>
-            );
-        })
-    );
+            if (val.field in row) {
+                DefinitionLists.push(<DL.Term>{label}</DL.Term>);
+                DefinitionLists.push(
+                    <DL.Description>
+                        {val.mapping
```

Comment:
`val?.mapping[row[val.field]] ? val.mapping[row[val.field]] : row[val.field]
`
can you rewrite this using one ternary operator?

Reply:
Updated with single ternary operation.